.azure-pipelines/steps/create-virtualenv.yml

```diff
@@ -5,7 +5,7 @@ steps:
         $env:PYTHON_EXE="python"
     }
     Write-Host "Found Python: $env:PYTHON_EXE"
-    Write-Host '##vso[task.setvariable variable=PY_EXE]$env:PYTHON_EXE'
+    Write-Host '##vso[task.setvariable variable=PY_EXE]"$env:PYTHON_EXE"'
```

Comment:
`PY_EXE` and `PYTHON_EXE` are not consistent.

Reply:
Yeah, I was attempting to figure out whether there was a conflict between setting it in the environment and having it set using the pipeline tooling as I was obviously struggling to get azure to play nicely here but it's sorted out and this isn't actually relevant anymore